CMakeLists.txt

```diff
@@ -547,6 +547,10 @@ if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU" AND CMAKE_CXX_COMPILER_VERSION VERS
     add_compile_options(-fdiagnostics-color=always)
 endif()
 
+# fold expression depth
+# device_grouped_conv2d_fwd_xdl_ngchw_gkyxc_ngkhw_f16_comp_instance.cpp
+add_compile_options(-fbracket-depth=3136)
```

Comment:
alternative is to implement some sort of paging for the parameter packs, not sure how to do it yet